test/Extension.tests/JobhostEndToEnd.cs

```diff
@@ -203,6 +220,11 @@ public void TestEventGridToNuget([EventGridTrigger] EventGridEvent value)
                 _functionOut = value.Subject;
             }
 
+            public void TestEventGridToCollection([EventGridTrigger] EventGridEvent[] value)
```

Comment:
Recommend adding tests for string[], JObject[] and other types

Reply:
@mathewc, oh so users are able to use string, JObject as well? That's good to know. I am guessing that's used for out of proc as well? Wondering what's the best way to add tests for out of proc as well.

Reply:
Yes, take a look at the existing tests. There are examples for binding to string, JObject, EventGridEvent and user defined Pocos. Your changes should support array versions of all of those.